app/src/main/java/com/standuptracker/RetrofitClientInstance.kt

```diff
@@ -6,12 +6,12 @@ import retrofit2.converter.gson.GsonConverterFactory
 object RetrofitClientInstance {
 
     private var retrofit: Retrofit? = null
-    private val BASE_URL = "https://api.npoint.io"
+    private const val BASE_URL = "https://api.npoint.io"
```

Comment:
It is best practice to mark a val as `const` if its value will never change

Reply:
yes, good.